docs/resources/identity_acl.md

```diff
@@ -0,0 +1,69 @@
+---
+subcategory: "Identity and Access Management (IAM)"
+---
+
+# huaweicloud\_identity\_acl
+
+The acl take effect only for IAM users under the Domain account, not the account itself.
+
+## Example Usage
+
+```hcl
+resource "huaweicloud_identity_acl" "acl" {
+  acl_type = "console"
+
+  ip_cidrs {
+    cidr        = "159.138.39.192/32"
+    description = "This is a test ip address"
+  }
+  ip_ranges {
+    range       = "0.0.0.0-255.255.255.0"
+    description = "This is a test ip range"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `acl_type` - (Required, String, ForceNew) Specifies the ACL is created through the console or API.
```

Comment:
suggest to rename it to `type`

Reply:
yeah, `type` is better than `acl_type`. Do you think `ip_cidrs` need to rename?